package.xml

```diff
@@ -60,14 +56,12 @@
   <depend>genmsg</depend>
   <depend>genpy</depend>
   <depend>geometry_msgs</depend>
-  <depend>glut</depend>
+  <depend>glut</depend> <!-- ??? -->
```

Comment:
What does the `<!-- ??? -->` comment suggest?

Reply:
I don't know. It was left around from previous work. Deleted.